include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -346,6 +347,7 @@ logging_resource_adaptor<Upstream> make_logging_adaptor(
  * @tparam Upstream Type of the upstream `device_memory_resource`.
  * @param upstream Pointer to the upstream resource
  * @param stream The ostream to write log info.
+ * @param auto_flush If true, flushes the log for every (de)allocation. Warning, this will degrade
```

Comment:
degrade... performance? (missing word)

Reply:
Yup forgot to copy the subsequent line over.